lib/matplotlib/_mathtext.py

```diff
@@ -2322,6 +2328,24 @@ def _make_space(self, percentage):
         r'\!':         -0.16667,  # -3/18 em = -3 mu
     }
 
+    _unit_multipliers = {
+         # pt|mm|cm|in|ex|em|mu|sp|bp|dd|pc|nc|nd|cc
+        "pt": 1,
+        'mm': 7227/2540,
+        'cm': 7227/254,
+        'in': 7227/100,
+        'ex': 35271/8192,
+        'em': 655361/65536,
```

Comment:
ex and em are probably not fixed values as they should(?) depend on the actual font being used, e.g. whether we're in a mathbf or a mathcal or whatnot.  OTOH we don't necessarily need to support them for now?